example/aws/functions-remote.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: pkg.crossplane.io/v1beta1
+kind: Function
+metadata:
+  name: function-shell
+#  annotations:
+#    # This tells crossplane beta render to connect to the function locally.
+#    render.crossplane.io/runtime: Development
+spec:
+  # This is ignored when using the Development runtime.
+  #package: xpkg.upbound.io/crossplane-contrib/function-shell:v0.3.0
+  package: index.docker.io/steve/function-shell:v0.3.0-bookworm-2
```

Comment:
We should update this to the crospslane-contrib repo v0.4.1 and cut a release. <https://github.com/crossplane-contrib/function-shell/pkgs/container/function-shell>